### ISSUE TITLE
Outdated Readme

### ISSUE THREAD

**Issue by awenger** [scrogson/hedwig#13]
```
Hedwig.start_client(x)
** (UndefinedFunctionError) undefined function: Hedwig.start_client/1
    (hedwig) Hedwig.start_client(%{})
```

**Comment by scrogson** [comment]
@awenger, sorry the README is actually up-to-date with the master branch. I should get a new release out soon. Sorry for the confusion.

**Comment by scrogson** [comment]
@awenger I have released `v0.2.0` which reflects the `README`.
